nautobot/core/templates/inc/nav_menu.html

```diff
@@ -88,43 +88,6 @@
                     </span>
                 </div>
             </form>
-            {% comment %}
-            The following if statement means the Query tab is only rendered when the GraphiQL
-            is rendered.
-            {% endcomment %}
-            {% if  graphiql and perms.extras.view_graphqlquery %}
```

Comment:
Does that mean we can now delete https://github.com/nautobot/nautobot/blob/develop/nautobot/core/views/__init__.py#L224 (`data["graphiql"] = True`) as part of this PR?